server/src/services/database.service.spec.ts

```diff
@@ -13,211 +12,170 @@ describe(DatabaseService.name, () => {
 
   beforeEach(() => {
     delete process.env.DB_SKIP_MIGRATIONS;
+    delete process.env.DB_VECTOR_EXTENSION;
     databaseMock = newDatabaseRepositoryMock();
     loggerMock = newLoggerRepositoryMock();
     sut = new DatabaseService(databaseMock, loggerMock);
+
+    databaseMock.getExtensionVersion.mockResolvedValue('0.2.0');
   });
 
   it('should work', () => {
     expect(sut).toBeDefined();
   });
 
-  describe.each([
-    [{ vectorExt: DatabaseExtension.VECTORS, extName: 'pgvecto.rs', minVersion: new Version(0, 1, 1) }],
-    [{ vectorExt: DatabaseExtension.VECTOR, extName: 'pgvector', minVersion: new Version(0, 5, 0) }],
-  ] as const)('init', ({ vectorExt, extName, minVersion }) => {
```

Comment:
The tests should still test everything with pgvector and pgvecto.rs. There's only one test for pgvector now from what I can see.

Reply:
This is a unit test not a e2e test. Can you tell me what code path in the database service is not tested anymore?

Reply:
The checks for updating, enforcing version constraints and displaying the right extension in logs aren’t being tested. The tests would still pass if we only handled them for pgvecto.rs.

Reply:
I can add tests to verify the extension in the log message and some to verify the semver ranges work as expected

Reply:
Alright, I've updated the tests to check semver and log messages for the extension name.

Reply:
Sweet, thanks!